src/databricks/labs/ucx/upgrades/v0.16.0_changing_cluster_table_schema.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=invalid-name
+
+import logging
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.lsql.backends import StatementExecutionBackend
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.ucx.config import WorkspaceConfig
+
+logger = logging.getLogger(__name__)
+
+
+def upgrade(installation: Installation, ws: WorkspaceClient):
+    config = installation.load(WorkspaceConfig)
+    sql = f"ALTER TABLE hive_metastore.{config.inventory_database}.clusters ADD COLUMNS(policy_id string,spark_version string)"
```

Comment:
Are these columns nullable? Did you try executing this statement on a database of the previous version?

Reply:
The columns are nullable. Ran this with the previous version of database.
<img width="1079" alt="image" src="https://github.com/databrickslabs/ucx/assets/127273819/6b36cf1d-38e3-4813-a501-c53bf70077c5">